test/definitions-eu/infra-agent/suse/suse154-infra.json

```diff
@@ -0,0 +1,32 @@
+{
+    "global_tags": {
+        "owning_team": "virtuoso",
+        "Environment": "development",
+        "Department": "product",
+        "Product": "virtuoso"
+    },
+
+    "resources": [{
+        "id": "host1",
+        "provider": "aws",
+        "type": "ec2",
+        "size": "t3.nano",
+        "ami_name": "suse-sles-15-sp4-v????????-hvm-*"
+    }],
+
+    "instrumentations": {
+      "resources": [
+        {
+            "id": "nr_infra",
+            "resource_ids": ["host1"],
+            "provider": "newrelic",
+            "source_repository": "https://github.com/newrelic/open-install-library",
+            "deploy_script_path": "test/deploy/linux/newrelic-cli/install-recipe/roles",
+            "params": {
+                "recipe_content_url": "https://raw.githubusercontent.com/newrelic/open-install-library/main/recipes/newrelic/infrastructure/suse.yml",
+                "validate_output": "New Relic installation complete"
```

Comment:
@rthorn-nr is this the correct validate_output to ensure infra installed successfully?

Reply:
We'd want `Infrastructure Agent\\s+\\(installed\\)`

Reply:
I corrected that: https://github.com/newrelic/open-install-library/pull/833/commits/694a4037bba823c83e30192c74c1e508d930752b